src/snapred/backend/dao/request/ReductionRequest.py

```diff
@@ -2,14 +2,12 @@
 
 from pydantic import BaseModel, ConfigDict
 
-from snapred.backend.dao.state.FocusGroup import FocusGroup
 from snapred.backend.error.ContinueWarning import ContinueWarning
 
 
 class ReductionRequest(BaseModel):
     runNumber: Union[str, List[str]]
     useLiteMode: bool
-    focusGroup: Union[Optional[FocusGroup], List[FocusGroup]]
```

Comment:
I think this is what caused all the failing tests.  I think the issue could be handled on the front-end, by filling in `focusGroup` from the grouping map instead.  It's already populated in the drop down.